### PR TITLE
Add ASV book CBMC examples

### PR DESCRIPTION
**Pull request by peterschrammel**
to make sure they actually work.

account.c with C10.desc currently doesn't work due to unsound pointer handling with concurrency.

- [X] Each commit message has a non-empty body, explaining why the change was made.
- [n/a] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [n/a] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [n/a] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [n/a] My commit message includes data points confirming performance improvements (if claimed).
- [X] My PR is restricted to a single feature or bugfix.
- [n/a] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Comment by NlightNFotis** [comment]
Hi Peter, what is the `asv book` referring to?

**Comment by martin-cs** [comment]
+1 would like to know, sounds interesting.

**Comment by peterschrammel** [comment]
An Automated Software Verification book consisting of chapters about all tools participating in SV-COMP.

File: regression/book-examples/account/C10.desc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+KNOWNBUG
```
Comment (tautschnig) [inline regression/book-examples/account/C10.desc:1]: Perhaps we should expose `allow-pointer-unsoundness` as a command-line option.

**Review by tautschnig — APPROVED** [review]

**Pull request merged**